dataikuapi/dss/savedmodel.py

```diff
@@ -150,6 +150,37 @@ def import_mlflow_version_from_path(self, version_id, path, code_env_name="INHER
         finally:
             shutil.rmtree(archive_temp_dir)
 
+    def import_mlflow_version_from_managed_folder(self, version_id, managed_folder, path, code_env_name="INHERIT"):
+        """
+        Create a new version for this saved model from a path containing a MLFlow model in a managed folder.
+
+        Requires the saved model to have been created using :meth:`dataikuapi.dss.project.DSSProject.create_mlflow_pyfunc_model`.
+
+        :param str version_id: Identifier of the version to create
+        :param str managed_folder: Identifier of the managed folder or `dataikuapi.dss.managedfolder.DSSManagedFolder`
+        :param str path: Path of the MLflow folder in the managed folder
+        :param str code_env_name: Name of the code env to use for this model version. The code env must contain at least
+                                  mlflow and the package(s) corresponding to the used MLFlow-compatible frameworks.
+                                  If value is "INHERIT", the default active code env of the project will be used
+        :return a :class:MLFlowVersionHandler in order to interact with the new MLFlow model version
+        """
+        # TODO: Add a check that it's indeed a MLFlow model folder
+        folder_ref = None
+        if managed_folder is not None:
+            if type(managed_folder) is DSSManagedFolder:
+                folder_ref = "{}.{}".format(managed_folder.project_key, managed_folder.id)
+            else:
+                folder_ref = managed_folder
+
+        self.client._perform_empty(
+            "POST", "/projects/{project_id}/savedmodels/{saved_model_id}/versions/{version_id}?codeEnvName={codeEnvName}".format(
+                project_id=self.project_key, saved_model_id=self.sm_id, version_id=version_id, codeEnvName=code_env_name
+            ),
+            params={"folderRef": folder_ref, "path": path},
+            files={"file": (None, None)}
```

Comment:
[Nitpick] Is this `files` argument required here?

Reply:
Indeed, `requests` does not allow setting "multipart" without a file argument (see this post: https://github.com/psf/requests/issues/1081). On the backend side, the request needs to be "multipart" even if the file parameter is optional.

Reply:
Alright. This could be noted in an inline comment to avoid accidental removal.

```suggestion
            files={"file": (None, None)}  # required for backend-mandated multipart request
```